acceptance/User.Specs/Features/SearchByOdsCode.feature

```diff
@@ -6,22 +6,20 @@ Scenario: SearchByOdsCodeWithResult
   Given I have opened the ODS search page
   When I enter a valid ODS code into the search box
   And I click the find button
-  Then I am taken to the results page
-  And there is 1 result
+  Then I am taken to the detail page
 
 Scenario: SearchByOdsCodeNoResult
   Given I have opened the ODS search page
-  When I enter a valid ODS code into the search box
+  When I enter a non-existent ODS code into the search box
   And I click the find button
-  Then I am taken to the results page
-  And there are 0 results
+  Then I am taken to the no results page
 
-Scenario: SearchByOdsCodeBadCode
-  Given I have opened the ODS search page
-  When I enter an invalid ODS code into the search box
-  And I click the find button
-  Then I remain on the ODS search page
-  And a validation error is shown
+# Scenario: SearchByOdsCodeBadCode
```

Comment:
We need to make sure that there is a story in the backlog for this work.

Reply:
Created
https://nhsd-jira.digital.nhs.uk/browse/GCNDSP-325